src/benchmarks/micro/README.md

```diff
@@ -75,16 +76,16 @@ We once again encourage you to read the [full docs about BenchmarkDotNet](../../
 
 ### Categories
 
-Every micro benchmark should belong to either CoreCLR, CoreFX or ThirdParty category. It allows for proper filtering for CI runs:
+Every micro benchmark should belong to either Runtime, Libraries or ThirdParty category. It allows for proper filtering for CI runs:
 
-* CoreCLR - benchmarks belonging to this category are executed for CoreCLR CI jobs
-* CoreFX - benchmarks belonging to this category are executed for CoreFX CI jobs
+* Runtime - benchmarks belonging to this category are executed for Runtime CI jobs
```

Comment:
It's worth noting that we don't actually do this. We run all the benchmarks on all CI jobs, regardless of what area of the code is being touched. Until we decide to change that perhaps we should remove this.

Reply:
I've removed it.